src/components/VoteScreen.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+import CandidateStatus from './CandidateStatus.js';
+
+export default function VoteScreen() {
+  const [candidates, setCandidates] = useState(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
```

Comment:
```suggestion
    const getCandidateList= async () => {
```

`fetchData`보단 후보자리스트를 받아오는 더 명확한 함수명인 `getCandidateList`가 좋을 것 같아요 :)